gamestonk_terminal/stocks/sector_industry_analysis/sia_controller.py

```diff
@@ -119,8 +125,148 @@ class SectorIndustryAnalysisController(BaseController):
         "ev": ("defaultKeyStatistics", "enterpriseValue"),
         "fpe": ("defaultKeyStatistics", "forwardPE"),
     }
-    mktcap_choices = ["Small", "Mid", "Large", "small", "mid", "large"]
+
+    visualise_choices = [
```

Comment:
In order to avoid duplicating stuff, i.e. if you change one side you need to change the other. Here you can declare this as something like: ```list(sa_keys["BS"].keys()) + list(sa_keys["CF"].keys()) + list(sa_keys["IS"].keys())```